src/components/BrewerySearch.js

```diff
@@ -1,12 +1,14 @@
 import React, { Component } from 'react'
-import PropTypes from 'prop-types'
 import Typed from 'react-typed'
 import RadioButtonContainer from './RadioButtonContainer'
+import SearchAlert from './SearchAlert.js'
+
 
 export default class BrewerySearch extends Component {
   state = {
     search: '',
-    searchFilter: 'city'
+    searchBy: 'city',
+    alertMessage:''
```

Comment:
alertMessage in this case should be a boolean value and not a string as it's a conditional, - it's either shown or it's hidden. It's initial state should be false as the Alert shouldn't be shown at the beginning.